lib/ansible/playbook/task.py

```diff
@@ -44,7 +45,7 @@
 display = Display()
 
 
-class Task(Base, Conditional, Taggable, Become):
+class Task(Base, Conditional, Taggable, Become, Collection):
```

Comment:
instead of importing the mixin in every object, does it makes sense to just add it to Base?

Reply:
There are some objects it (arguably) doesn't make sense or function on, like task_include, etc, so putting it in Base could make it succeed in places it shouldn't (we have this problem already on a few others IIRC).